templates/glue/rte/templates.yaml

```diff
@@ -0,0 +1,66 @@
+dataset: glue
+subset: rte
+templates:
+  03a7ae07-5ddd-46c4-92f3-2152223d44ec: !Template
+    id: 03a7ae07-5ddd-46c4-92f3-2152223d44ec
+    jinja: '{{sentence1}}
+
+      Does this mean that "{{sentence2}}" is true? {{"A) yes or B) no."}}
+
+      |||
+
+      {{["yes", "no"][label]}}'
+    name: mean
+    reference: ''
+  4ee6ff27-de63-4e7b-a9d4-82a17eba407a: !Template
+    id: 4ee6ff27-de63-4e7b-a9d4-82a17eba407a
+    jinja: 'Does "{{sentence2}}" imply that "{{sentence1}}"? Please answer either
+      {{"yes"}} or {{"no"}}.
+
+      |||
+
+      {{["yes", "no"][label]}}'
+    name: imply swapped
+    reference: ''
+  9e2b4267-ec23-44c8-b82a-107e2c890fec: !Template
+    id: 9e2b4267-ec23-44c8-b82a-107e2c890fec
+    jinja: 'We say that one sentence "{{"entails"}}" another sentence when the first
+      sentence implies the second sentence is true. Consider the following two sentences:
+
+      {{sentence1}}
+
+      {{sentence2}}
+
+      Is the relationship between these two sentences "{{"entailment"}}" or "{{"not
+      entailment"}}"?
+
+      |||
+
+      {{["entailment", "not entailment"][label]}}'
+    name: entailment explained
+    reference: ''
+  c8dfc879-40f2-412d-be1e-4cd70107f6e6: !Template
+    id: c8dfc879-40f2-412d-be1e-4cd70107f6e6
+    jinja: 'Does "{{sentence1}}" imply that "{{sentence2}}"? Please answer either
+      {{"yes"}} or {{"no"}}.
+
+      |||
+
+      {{["yes", "no"][label]}}'
+    name: imply
+    reference: ''
+  f56ffced-9b16-431a-8a17-501e63cddf73: !Template
+    id: f56ffced-9b16-431a-8a17-501e63cddf73
+    jinja: '{{sentence2}}
+
+      Does this imply
+
+      {{sentence1}}
```

Comment:
Does this imply {{sentence1}} *?*

Reply:
This would make the formatting weird because RTE sentences can have terminal punctuation marks themselves, e.g. it would look like
```
Weapons of Mass Destruction Found in Iraq.
Does this imply
No Weapons of Mass Destruction Found in Iraq Yet. ?
Please answer A) yes or B) no.